mypy/treetransform.py

```diff
@@ -67,12 +67,38 @@ def visit_import_from(self, node: ImportFrom) -> Node:
     def visit_import_all(self, node: ImportAll) -> Node:
         return ImportAll(node.id, node.relative)
 
+    def copy_argument(self, argument: Argument) -> Argument:
+        init_stmt = None  # type: AssignmentStmt
+
+        if argument.initialization_statement:
+            init_lvalue = cast(
+                NameExpr,
+                self.node(argument.initialization_statement.lvalues[0]),
+            )
+            init_lvalue.set_line(argument.line)
+            init_stmt = AssignmentStmt(
```

Comment:
It seems to me that we can always construct the new init statement in `Argument`. If nothing breaks when you do it it's probably fine. What do you think?